src/window_handle.rs

```diff
@@ -318,7 +321,9 @@ impl WindowHandle {
                 }
             }
         }
-        if was_focused != cx.app_state.focus {
+        if (was_focused != cx.app_state.focus) && !handled {
```

Comment:
I feel like this is not needed because the PointerDown event propagation was stopped. Can you please test if it fixes the issue without this check?

Reply:
This is the part I don't completely understand what is happening. If I remove that check then anytime that an event returns `Stop` (even a different event handler such as PointerDown) the focus will be switched away.